src/utils/collection-stream.js

```diff
@@ -169,13 +169,16 @@ export const createCollectionWriteStream = function(
 export const createReadableCollectionStream = function(
   dataService,
   ns,
-  spec = { filter: {} }
+  spec = { filter: {} },
+  projection = {}
 ) {
   const { project, limit, skip } = spec;
+
   return dataService
     .fetch(ns, spec.filter || {}, {
       explicitlyIgnoreSession: true,
       project,
+      projection,
```

Comment:
Can the driver accept `project` and `projection` options?

Reply:
Hm. What is the project option meant to do, in fact? I don't see it under [find](https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#find), is dataService.fetch a wrapper around something else that I am missing?

Reply:
Ah looks like `project` and `fields` are deprecated in favor of `projection` so we should standardize on that as well